src/views/v2/commerce/checkout/order-modifiers/coupons.php

```diff
@@ -41,6 +46,11 @@
 
 ?>
 <div class="tec-tickets__commerce-checkout-coupons-wrapper tribe-tickets__form tribe-common-b2">
+	<p <?php tribe_classes( $add_coupon_link_classes ); ?>>
```

Comment:
Question: Why a paragraph tag? Are we just using it for the spacing?
A button isn't really _text_
This is the only `<p>` tag that isn't wrapping pure text.

It's not invalid HTML, but is it semantic?

Reply:
I originally had the button as `<a>` and then realized a `<button>` would be better. I just didn't drop the `<p>` tags around it, but I'll do that now.